### PR TITLE
[#4] Splits docker-compose file into components, rewrites bash script

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#4]

- [x] Splits the `docker-compose.yml` file into a `base` and `dev` file. This allows for creation of specific environments for dev, CI, and production.
- [x] Rewrites our script to manage our application. At the moment, most of the functionality is delegated to the `docker-compose` tool.

### CODE REVIEW

**Review by dodoels — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ To install Docker and Docker Compose, you can follow the instructions [here](htt
...
 ```
 
 In order to shut down all containers:
 
 ```
-docker-compose down
+./cjl down
 ```
 
 ## Running or Testing Specific Services
```
Comment (dodoels) [inline README.md:32]: `[Web Client](src/web_app/README.md)` change `web_app` to `web_client`

**Review by dodoels — CHANGES_REQUESTED** [review]

File: cjl [hunk 2]
```diff
@@ -1,108 +1,33 @@
...
+  exit
+fi
+
+# Choose an environment, default is dev
+if [ -z "$COMPOSE_FILE" ]; then
+    COMPOSE_FILE='dev'
+fi
+
+# Passes script arguments to docker-compose
+COMPOSE="docker-compose -f docker-compose.base.yml -f docker-compose.$COMPOSE_FILE.yml $@"
```
Comment (arekmano) [inline cjl:31]: much cleaner :+1:

**Review by arekmano — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lancelafontaine) [reply README.md:32]: Good catch

**Review by dodoels — APPROVED** [review]

**Pull request merged**